bookkeeper-server/src/main/java/org/apache/bookkeeper/replication/Auditor.java

```diff
@@ -743,11 +825,11 @@ public void operationComplete(int rc, Set<LedgerFragment> fragments) {
      * be run very often.
      */
     void checkAllLedgers() throws BKException, IOException, InterruptedException, KeeperException {
-        final BookKeeper client = createBookKeeperClient(conf);
-        final BookKeeperAdmin admin = new BookKeeperAdmin(client, statsLogger);
+        final BookKeeper localClient = createBookKeeperClient(conf);
```

Comment:
Better to use try-with-resources otherwise you could fail to close the client if the admin close fails

Reply:
yeah..but it is existing code, probably it would be better to do it in different commit. Otherwise it shows as if complete code is changed.

Reply:
Ok